Jetcaster/app/src/main/res/values/themes.xml

```diff
@@ -16,7 +16,7 @@
   -->
 <resources>
 
-    <style name="Theme.Jetcaster" parent="Theme.MaterialComponents.NoActionBar">
+    <style name="Theme.Jetcaster" parent="android:Theme.Material.NoActionBar">
         <item name="colorPrimary">#ff00ff</item>
```

Comment:
Where do these attrs come from? Shouldn't they have android ns?

Reply:
Hmmm, no idea how this worked for me. Fixed.